test_utils/mockHttpCalls.js

```diff
@@ -53,42 +54,39 @@ export const infoResponse = {
   workers: {
     count_per_cluster: { max: null, default: 3 },
     instance_type: {
-      options: [
-        'c5.large',
-        'c5.xlarge',
-        'c5.2xlarge',
-        'c5.4xlarge',
-        'c5.9xlarge',
-        'i3.xlarge',
-        'm3.large',
-        'm3.xlarge',
-        'm3.2xlarge',
-        'm4.large',
-        'm4.xlarge',
-        'm4.2xlarge',
-        'm4.4xlarge',
-        'm5.large',
-        'm5.xlarge',
-        'm5.2xlarge',
-        'm5.4xlarge',
-        'r3.large',
-        'r3.xlarge',
-        'r3.2xlarge',
-        'r3.4xlarge',
-        'r3.8xlarge',
-        'r5.2xlarge',
-        't2.large',
-        't2.xlarge',
-        't2.2xlarge',
-        'p2.xlarge',
-        'p3.2xlarge',
-        'p3.8xlarge',
-      ],
+      options: ['m4.xlarge'],
       default: 'm4.xlarge',
     },
   },
 };
 
+export const azureInfoResponse = {
+  general: {
+    availability_zones: { default: 1, max: 1 },
+    datacenter: 'westeurope',
+    installation_name: 'ghost',
+    provider: Providers.AZURE,
```

Comment:
Since this is technically out from the API, I don't know if we need the Providers.AZURE thing here.

That said, I don't mind.

But I would like to keep it the same between the types of calls. So in `KVMINfoResponse` we use just `kvm` instead of `Providers.KVM`. Can we align this here so it's the same?

Also noticed further down `Providers.azure` a few times (no uppercase).

https://github.com/giantswarm/happa/pull/879/files#diff-f5234f59c0885943656b6bba42845e98R403

Reply:
Changed, thanks!